app/src/main/res/values/strings.xml

```diff
@@ -33,6 +33,7 @@
     <string name="no_search_results">Hmmm, we\'\re not getting\nany results. Our bad - try\nanother search</string>
     <string name="event_details">Event details</string>
     <string name="see_maps">See maps</string>
+    <string name="error_message">There was an error. Tab here to try again.</string>
```

Comment:
Check whether the message is similar in EventBrite

Reply:
Yes, it is similar in the updated version of EventBrite.

Reply:
Could you please attach a screenshot of how it looks?

Reply:
> Could you please attach a screenshot of how it looks?

Uploaded in PR description.